### DISCUSSION TITLE
Using open collective

### DISCUSSION THREAD

**Discussion by muxina-credo** [FossifyOrg/General-Discussion#51] [Ideas]
Hi.  I was wanted to suggest that using [Open Collective](https://opencollective.com/) my earn you more donations. I guess if people know how underfunded you are might be more willing to help this project.

Good luck!

**Comment by Aga-C** [comment]
It's discussed here: https://github.com/orgs/FossifyOrg/discussions/14#discussioncomment-7752674.
